problemsets-mat223/linearalbegra.tex

```diff
@@ -2289,13 +2314,57 @@ \section*{Subspaces and Bases}
 	For each set, draw it and explain whether or not it is a subspace of $\R^2$.
 	\begin{parts}
 		\item $A=\{\vec x\in\R^2:\vec x=\mat{a\\0}\text{ for some }a\in\Z\}$.
+			\begin{solution}
+				XXXX
+
+				$A$ is not a subspace, since for example $\mat{1\\0} \in A$ but 
+				$\frac{1}{2}\mat{1\\0}\notin A$. 
+			\end{solution}
 		\item $B=\{\vec x\in\R^2:\vec x\neq \mat{0\\0}\}$.
+			\begin{solution}
+				XXXX
+					
+				$B$ is not a subspace, since for example $\mat{1\\0}$ and $\mat{-1\\0}$
+				are both in $B$, but their sum is $\mat{0\\0}$ which is not in $B$.
+			\end{solution}
 		\item $C=\{\vec x\in\R^2:\vec x=\mat{0\\t}\text{ for some }t\in\R\}$.
+			\begin{solution}
+				XXXX
+					
+				$C$ is a subspace. 
```

Comment:
Since this is the first time they're encountering a real proof, I'd like to spell it out. It can be terse, but I don't want to skip steps.